### PR TITLE
Format code hello triangle

### PR DESCRIPTION
**Pull request by Mautar55**
GL.BindBuffer(BufferTarget.ArrayBuffer, VertexBufferObject);

GL.BufferData(BufferTarget.ArrayBuffer, vertices.Length * sizeof(float), vertices, BufferUsageHint.StaticDraw);

Those are now are in the proper frame.

### CODE REVIEW

File: learn/chapter1/2-hello-triangle.md [hunk 1]
```diff
@@ -120,11 +120,15 @@ VertexBufferObject = GL.GenBuffer();
...
 OpenGL has many types of buffer objects and the buffer type of a vertex buffer object is `BufferTarget.ArrayBuffer`. OpenGL allows us to bind to several buffers at once as long as they have a different buffer type. We can bind the newly created buffer to the `BufferTarget.ArrayBuffer` target with the `GL.BindBuffer` function:
 
+```cs
 `GL.BindBuffer(BufferTarget.ArrayBuffer, VertexBufferObject);`
+```
 
 From that point on any buffer calls we make (on the `BufferTarget.ArrayBuffer` target) will be used to configure the currently bound buffer, which is VertexBufferObject. Then we can make a call to `GL.BufferData` function that copies the previously defined vertex data into the buffer's memory:
 
+```cs
 `GL.BufferData(BufferTarget.ArrayBuffer, vertices.Length * sizeof(float), vertices, BufferUsageHint.StaticDraw);`
```
Comment (frederikja163) [inline learn/chapter1/2-hello-triangle.md:130]: Remove the code blocks from this line before we can merge

File: learn/chapter1/2-hello-triangle.md [hunk 2]
```diff
@@ -120,11 +120,15 @@ VertexBufferObject = GL.GenBuffer();
 
 OpenGL has many types of buffer objects and the buffer type of a vertex buffer object is `BufferTarget.ArrayBuffer`. OpenGL allows us to bind to several buffers at once as long as they have a different buffer type. We can bind the newly created buffer to the `BufferTarget.ArrayBuffer` target with the `GL.BindBuffer` function:
 
+```cs
 `GL.BindBuffer(BufferTarget.ArrayBuffer, VertexBufferObject);`
```
Comment (frederikja163) [inline learn/chapter1/2-hello-triangle.md:124]: Remove the code blocks from this line before we can merge

**Review by frederikja163 — CHANGES_REQUESTED** [review]
You need to remove the old code blocks that were there before im gonna merge this.

**Comment by NogginBops** [comment]
Thanks for the PR @Mautar55!
Sorry for letting it go unmerged for so long. :)

**Pull request merged**